src/sync/commits.ts

```diff
@@ -26,14 +27,23 @@ export const getCommitTask = async (
 	logger: LoggerWithTarget,
 	_github: GitHubAPI,
 	gitHubClient: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
 	cursor?: string | number,
 	perPage?: number): Promise<CommitData> => {
-	logger.info("Syncing commits: started");
+	const itemNumber = Number(cursor?.toString().split(" ").pop() || 0);
```

Comment:
whats going on here?
`const itemNumber = Number(cursor?.toString().split(" ").pop() || 0);`
I would expect the cursor to look like **Y3VycOAKYOHs4Apg4e** since its from graphQL?

Reply:
The commit cursor is unique here. It's in the format of `6ee0a9e7b43033ca54f3e313c32017e05182c691 3107`, the number at the end is actually the amount of commits down from the HEAD.  Not the best way to do this, I'll agree, since GH could change their cursor format tomorrow and break everything, but this is temporary until we change how we're processing syncs.

Reply:
Shouldn't this be covered/described in a test? (And similar things?)

Reply:
tests added